### PR TITLE
Ignore PyPy numpy errors

### LINKED ISSUE #2076: Numpy is broken on pypy
Looks like Travis have updated PyPy from 2.6.1 ([working build](https://travis-ci.org/python-pillow/Pillow/jobs/150329419)) 5.3.1 ([faild build](https://travis-ci.org/python-pillow/Pillow/jobs/152014515)). In the 2.6.1 the `TestNumpy.test_16bit` test used to be skipped due to lask of numpy.

```
======================================================================
ERROR: TestNumpy.test_16bit
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/python-pillow/Pillow/Tests/test_numpy.py", line 117, in test_16bit
    self._test_img_equals_nparray(img, np_img)
  File "/home/travis/build/python-pillow/Pillow/Tests/test_numpy.py", line 107, in _test_img_equals_nparray
    np_size = np.shape[1], np.shape[0]
IndexError: tuple index out of range
----------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by arjennienhuis**
Fixes #2076

Just ignore the numpy tests on PyPy

### CODE REVIEW

File: Tests/test_numpy.py [hunk 1]
```diff
@@ -104,6 +107,7 @@ def test_3d_array(self):
         self.assert_image(Image.fromarray(a[:, :, 1]), "L", TEST_IMAGE_SIZE)
 
     def _test_img_equals_nparray(self, img, np):
+        self.assertGreaterEqual(len(np.shape), 2)
```
Comment (hugovk) [inline Tests/test_numpy.py]: Is this needed?

**Comment by hugovk** [comment]
This is ok for short term, but it'd be better to find out why PyPy 2.6.1 worked but PyPy 5.3.1 fails, and report back to PyPy.

File: Tests/test_numpy.py (hunk 1 above)
Comment (arjennienhuis) [inline Tests/test_numpy.py]: this is why the test crashes instead of fails

**Comment by arjennienhuis** [comment]
pypy numpy simply isn't finished:

http://buildbot.pypy.org/numpy-status/latest.html

Travis used to skip the numpy builds for pypy, because numpy wasn't available.

If you know of  a working configuration of pypy+numpy I'd like to find out what changed. Can you tell me the versions of both?

**Pull request merged**